litellm/llms/custom_httpx/llm_http_handler.py

```diff
@@ -717,10 +718,7 @@ def rerank(
         api_base: Optional[str] = None,
         client: Optional[Union[HTTPHandler, AsyncHTTPHandler]] = None,
     ) -> RerankResponse:
-
-        provider_config = ProviderConfigManager.get_provider_rerank_config(
-            model=model, provider=litellm.LlmProviders(custom_llm_provider)
-        )
```

Comment:
Deleted the creation of the provider config here because we already created it earlier. Opted to pass it in to this method instead